guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -92,11 +92,13 @@ ifdef::satellite[]
 # yum clean metadata
 ----
 +
-. Ensure that the `rubygem-foreman_maintain` package that provides `{foreman-maintain}` is installed and up to date:
+. The `rubygem-foreman_maintain` is installed from the {Project} {ProjectVersion} maintenance repository or upgraded from the {Project} {ProjectVersion} maintenance repository if currently installed.
++
+Ensure {SmartProxy} has access to repository `{RHEL}-7-server-satellite-maintenance-{ProjectVersion}-rpms` and execute:
 +
 [options="nowrap" subs="attributes"]
 ----
-# {package-install} rubygem-foreman_maintain
+# {project-context}-maintain self-upgrade
```

Comment:
Please make an effort to check does an attribute exist before you introduce bizarre hybrids. 
This would print as "smart-proxy-maintain" or "capsule-maintain" in some contexts as you have it now, and that makes no sense. 

guides/common/attributes-satellite.adoc::foreman-maintain: satellite-maintain